packages/assertion-monitor/chains.ts

```diff
@@ -0,0 +1,63 @@
+import { ChildNetwork } from 'utils'
+import { Chain } from 'viem'
+import {
+  mainnet,
+  arbitrum,
+  arbitrumNova,
+  base,
+  sepolia,
+  holesky,
+  arbitrumSepolia,
+  baseSepolia,
+} from 'viem/chains'
+
+export const DEFAULT_TIMESPAN_SECONDS = 60 * 60 * 24 * 7 // 1 week
+
+export const supportedParentChains = [
+  mainnet,
+  arbitrum,
+  arbitrumNova,
+  base,
+
+  sepolia,
+  holesky,
+  arbitrumSepolia,
+  baseSepolia,
+]
+
+export const getChainFromId = (chainId: number): Chain => {
+  const chain = supportedParentChains.filter(chain => chain.id === chainId)
+  return chain[0] ?? null
+}
+
+export const getDefaultBlockRange = (chain: Chain): bigint => {
+  switch (chain) {
+    case mainnet:
+    case sepolia:
+    case holesky:
+      return BigInt(DEFAULT_TIMESPAN_SECONDS / 12)
+
+    case base:
+    case baseSepolia:
+      return BigInt(DEFAULT_TIMESPAN_SECONDS / 2)
+
+    case arbitrum:
+    case arbitrumNova:
+    case arbitrumSepolia:
+      return BigInt(DEFAULT_TIMESPAN_SECONDS / 12)
```

Comment:
Arbitrum produces up to 4 blocks per second actually (every 250ms, provided there's activity in the chain). It's Ethereum the one that produces 12 blocks per seconds.

Some Arbitrum block calculations are done in "Ethereum blocktime", that's why it might be a bit confusing. But in this case, you're searching for logs in a range of blocks in an Arbitrum chain (the parent chain of a specific L3), so 4 blocks per second should be assumed.